src/applications/personalization/account/containers/AccountApp.jsx

```diff
@@ -3,17 +3,9 @@ import { connect } from 'react-redux';
 import backendServices from 'platform/user/profile/constants/backendServices';
 import { selectUser, isLOA3 } from 'platform/user/selectors';
 
-import AccountMain from '../components/AccountMain';
```

Comment:
Pretty sure we can remove this entire component now since there is a 301 redirect that prevents people from ever going to the HTML that holds this component.

https://github.com/department-of-veterans-affairs/devops/blob/5a37c2865d9e82a34f5dd1e7a455679cc67710ef/ansible/deployment/config/revproxy-vagov/vars/redirects.yml#L483-L488

Reply:
Yup, removed in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/14525/files)